.github/mergify.yml

```
@@ -0,0 +1,7 @@
+pull_request_rules:
+  - name: Automatic merge for Dependabot pull requests
+    conditions:
+      - author=dependabot[bot]
+    actions:
+      merge:
+        method: squash
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)